xml/docu_styleguide.smartdocs.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       xmlns:its="http://www.w3.org/2005/11/its" version="5.0" xml:id="sec-smartdocs">
+
+ <title>Working with Smart Docs</title>
+ <para>
+  Smart Docs represent the modular approach to documentation whose goal is producing sets of
+  adaptable, solution-based, easy-to-find and reusable documents.
+ </para>
+ <para>
+  Smart Docs consist of smaller information units with the following characteristics:
+ </para>
+<itemizedlist>
+  <listitem>
+   <formalpara>
+    <title>Self-contained</title>
+    <para>
+     Each information unit stands for itself, including requirements, context, instructions, troubleshooting and FAQs.
+    </para>
+   </formalpara>
+  </listitem>
+  <listitem>
+   <formalpara>
+    <title>Cover one distinct topic</title>
+    <para>
+     Rather than covering a complete technology or a set of problems, each information unit
+     serves one distinct purpose. Five different types of information units are provided:
+    </para>
+    </formalpara>
+    <itemizedlist>
+    <listitem>
+     <para>
+      The assembly file builds the actual article from the components listed below.
+     </para>
+    </listitem>
```

Comment:
I think, listing the assembly here as a "types of information unit" is a bit odd and confusing.

From the "classic" view, types are just task, concept, reference, and glue. Assemblies are just a DocBook specific organization method to combine or group all the of the above types into a coherent documentation. Mixing both concepts/ideas (general and DocBook specific) is IMHO misleading. I wouldn't count it as a "type" per se and as such I wouldn't include it into this list.

If you ask me, listing all the types is fine. But I would move the entry about assemblies _after_ the list and mention it explicitly as an organization method.

---

Regarding the wording: The assembly doesn't "build" the actual article. It's just a means to organizing the mess. :wink: The build part is done by a tool chain (in our case it's daps). The verb implies that assemblies does an active part. It doesn't.

I would suggest:

```
The assembly file is an organizational unit to group the components listed above and put them in order.
```